scripts/_resources/r_god.gd

```diff
@@ -0,0 +1,14 @@
+extends Resource
+class_name RGod
```

Comment:
This is a good first start but we will probably need to iterate on what favors, tithes, and passive actually looks like in data.